### DISCUSSION TITLE
Does this library uses OSRM ?

### DISCUSSION THREAD

**Discussion by mziadd** [liodali/osm_flutter#394] [Q&A]
I want to make sure this library uses OSRM for routing, if so how i can use it ?

**Comment by liodali** [comment]
can you provide us more information because we are using api to draw roads in general internally 
or you can use your own api ( built on the top of osrm and use drawRoadManually )

**Reply by mziadd** [reply]
Is the API you are using built on top of osrm ?, And it sufficient to use it? does it have the features of osrm ?

**Reply by liodali** [reply]
we're cliend for opensource osrm api 
we still miss some fonctionality but for getting road is fully fonctional feel free to use and request any bug or needed  feature
